sdk/python/requirements.txt

```diff
@@ -0,0 +1,12 @@
+urllib3>=1.15,<1.25 #Fixing the version conflict with the "requests" package caused by the kubernetes package
```

Comment:
If the requirements are extracted, maybe also reference to this file in setup.py?

Reply:
It's a bit non-trivial. It might sound strange, but there is no built-in requirements.txt parser and also `setup` does not have full support by design.

This problem is not related to docs, so we can try addressing it in another PR.

Reply:
Sure. Let's put a TODO here as a reminder.